Open Judge System/Web/OJS.Web/App_GlobalResources/Areas/Contests/Views/Submissions/SubmissionsView.bg.resx

```diff
@@ -219,4 +219,7 @@
   <data name="Your_output" xml:space="preserve">
     <value>Твоят изход:</value>
   </data>
+  <data name="User_retest_prompt" xml:space="preserve">
+    <value>Решението ({0}/{1}), което сте изпратили на тази задача е загубило давност, заради промяна във входно-изходните данни от страна на администратор.&lt;b&gt; С кликането на бутона по-долу имате възможност да преизпратите решението на ново&lt;/b&gt;. Това ще доведе до ново оценяване и съответно обратна връзка за всеки един от предварително дефинираните входни данни (тестове). Имайте предвид, че е възможно решението да произведе различен брой точки от първоначално получените, заради гореспоменатата промяна в данните. Натиснете "Retest" за да изпратите отново решението!</value>
```

Comment:
`съответно обратна връзка за всеки едни от предварително дефинираните входни данни` -> `всеки едни`, not `всеки един`, Because we are talking about `данни`